packages/api-server/api_server/routes/tasks/test_scheduled_tasks.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=too-many-lines
```

Comment:
Don't disable the linter without a valid reason. I think the line count is probably due to duplicated code, see if it is possible to refactor them. If that is still an issue, then disable the rule globally (might want to do it regardless, imo this rule is not useful at all).

Reply:
Hmm, but this is only for testing though. We could refactor it and re-used code between tests, but I was trying to prevent any stack tracing, even if the checks are repeated over tests, it doesn't make much sense to have the errors on the same lines across tests.

Reply:
I think it is really up for debate whether code duplication is justified to make test more clear. I don't really have any strong opinions, if you think that it is ok, then just disable the lint globally. imo that is not a useful rule anyway, regardless of the reasons.